braindecode/models/sleep_stager_blanco_2020.py

```diff
@@ -50,7 +51,7 @@ class SleepStagerBlanco2020(nn.Module):
 
     def __init__(self, n_channels, sfreq, n_conv_chans=20, input_size_s=30,
                  n_classes=5, n_groups=2, max_pool_size=2, dropout=0.5, apply_batch_norm=False,
-                 return_feats=False):
+                 layers_returned="classifier"):
```

Comment:
@tgnassou you cannot do this. This will break backward compatibility. Please avoid changes that can break existing code.

Reply:
Even if the code becomes heavy ? (In general case)

Reply:
it's ok to change a public API but it needs some deprecation messages. See for example https://github.com/braindecode/braindecode/pull/413

now before moving forward with this you need to see with others if the API change is ok for the core devs of the project.

besides for me the term "features" is not ideal. For me the word "embedding" is more common to refer to the vector representation before the final layer.